app/assets/stylesheets/active_admin/mixins/_variables.scss

```diff
@@ -2,33 +2,236 @@
+// $form-label-color              -> REMOVED
+// $page-header-text-color        -> REMOVED
+// $border-width                  -> REMOVED
+// $cell-padding                  -> $table-cell-padding
+// $cell-horizontal-padding       -> REMOVED
+// $section-padding               -> $panel-body-padding
+// $text-input-horizontal-padding -> REMOVED
+// $text-input-total-padding      -> REMOVED
+// $blank-slate-border            -> REMOVED
+
+
+// Color theme from Flat UI (color swatch section at http://designmodo.github.io/Flat-UI/)
+
+
+//== Scaffolding
+
+$primary-color: #34495e !default;
+$success-color: #2ecc71 !default;
+$warning-color: #f1c40f !default;
+$error-color: #e74c3c !default;
+
+$body-background-color: #fff !default;
+
+$text-color: #3f4c4d !default;
+$text-muted-color: #7f8c8d !default;
+
+$link-color: #2980b9 !default;
+$link-hover-color: darken(#2980b9, 5%) !default;
+$link-hover-decoration: underline !default;
+
+//== Typography
+
+$font-family-sans-serif: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif !default;
+$font-family-serif: Georgia, Cambria, "Times New Roman", Times, serif !default;
+$font-family-monospace: Menlo, Monaco, Consolas, "Courier New", monospace !default;
+$font-family-base: $font-family-sans-serif !default;
+
+$font-weight-normal: 400 !default;
+$font-weight-semibold: 400 !default;
+$font-weight-bold: 700 !default;
+
+$font-size-base: 13px !default;
+
+$line-height-base: 1.5 !default;
+
+$body-font-family: $font-family-base !default;
+$body-font-size: $font-size-base !default;
+$body-font-weight: $font-weight-normal !default;
+
+$headings-font-family: inherit !default;
+$headings-font-weight: inherit !default;
+$headings-color: inherit !default;
+
+//== Components
+
+$border-radius-base: 0.25em !default;
+
+$padding-base-vertical: 6px !default;
+$padding-base-horizontal: 12px !default;
+
+$horizontal-page-margin: 20px !default;
+
 $sidebar-width: 270px !default;
-$cell-padding: 5px 10px 3px 10px !default;
-$cell-horizontal-padding: 12px !default;
-$section-padding: 15px !default;
-$text-input-horizontal-padding: 10px !default;
-$text-input-total-padding: $text-input-horizontal-padding * 2 + $border-width * 2;
 
-$blank-slate-border: 1px dashed #DADADA !default;
+//== z-indexes
+
+$z-index-dropdown: 1000 !default;
+$z-index-navbar: 1000 !default;
+$z-index-modal-backdrop: 1001 !default;
+$z-index-modal: 1002 !default;
+
+//== Blank Slate
+
+$blank-slate-color: $text-muted-color !default;
+
+//== Buttons
+
+$button-font-weight: $font-weight-bold !default;
+
+$button-default-color: $text-color !default;
+$button-default-background-color: #fff !default;
+$button-default-border-color: #d9d9d9 !default;
+
+$button-primary-color: #fff !default;
+$button-primary-background-color: $primary-color !default;
+$button-primary-border-color: darken($primary-color, 5%) !default;
+
+//== Forms
+
+$input-background-color: #fff !default;
+$input-border-color: lighten(#bdc3c7, 5%) !default;
```

Comment:
Color literals like `#bdc3c7` should only be used in variable declarations; they should be referred to via variable everywhere else.